src/fhir/PlanDefinition_cove.json

```diff
@@ -0,0 +1,438 @@
+    },
+    {
+      "definitionCanonical": "#Day371Msg1"
+    },
+    {
+      "definitionCanonical": "#NewYearsMsg"
+    },
+    {
+      "definitionCanonical": "#VeteransDayMsg"
+    },
+    {
+      "definitionCanonical": "#BirthdayMsg",
+      "trigger": [
+        {
+          "type": "named-event",
+          "name": "birthday"
+        }
+      ]
+    }
+  ],
+  "contained": [
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day14Msg0",
+      "name": "Day 14 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 14,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "18:05:00"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Hi {name}, I just wanted to thank you for participating in this program. It takes a lot of courage to share your story and we appreciate that you did. You are not alone on this journey. I am here to support you, and just want to let you know that my team and I are available Mon to Fri 9-5 PT, so I won’t see messages sent outside of those hours right away.  Please feel welcome (but never pressured) to reply or share anything on your mind. I'm so glad we have this chance to connect!  -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day14Msg1",
+      "name": "Day 14 Message 1",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 14,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "18:05:45"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "If you need immediate help, don't forget about the 24/7 National Hotline - 988 & press 1, or you can text the Veterans Crisis Line @ 838-255, or check our website: wefacethefight.org"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day43Msg0",
+      "name": "Day 43 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 43,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "10:19:25"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Hey, this is Riley just checking in. I hope you are doing okay! Feel free to let me know how things are going if it helps."
```

Comment:
@mcjustin not sure if `Riley` here is meant to be the author name here, i.e. `{userName}`?

Reply:
The COVE team are using the pseudonym "Riley" instead of their real names.